packages/cli/src/lib/new/factories/backendPlugin.ts

```diff
@@ -78,6 +78,31 @@ export const backendPlugin = createFactory<Options>({
           },
         );
       });
+
+      await Task.forItem('backend', 'adding plugin', async () => {
```

Comment:
This seems worth breaking out into a reusable function `addBackendAddLine` next to `addPackageDependency` or so, instead of copying it around